### DISCUSSION TITLE
oppo watch4 can't work with dexcom G6

### DISCUSSION THREAD

**Discussion by SimonXuku** [NightscoutFoundation/xDrip#3264] [General]
oppo watch4 is a full version of android with 2G ram and esim, it can run AAPS standalone ,I can use AAPS with libre2 and dash over 20hrs,so my son no need to take the phone when he is in school, . I tested libre 2 , libre3 by juggluco,it works.but when I use dexcom G6 , it can not connect the anubis transmitter,the transmitter is tested. the BYOD dexcom G6 app also can't work.I don't know how to resolve it.

![juggluco](https://github.com/NightscoutFoundation/xDrip/assets/111485771/6bf5f3a5-6306-4444-b83c-c0a97d873b18)
![xdrip](https://github.com/NightscoutFoundation/xDrip/assets/111485771/5c95bd9d-0bdc-4267-b5bc-225196f70c00)
![xdrip2](https://github.com/NightscoutFoundation/xDrip/assets/111485771/9dadb004-876f-46d8-9429-8b3d4fb50ce2)

**Comment by Navid200** [comment]
Please provide a link to the watch, where it can be seen that it is indeed Android and what version.

**Reply by SimonXuku** [reply]
it's a modified android 11 in China, wearos for international.


![image](https://github.com/NightscoutFoundation/xDrip/assets/111485771/4ff4b92f-33c3-4468-b7f1-68f1e36462af)
![image](https://github.com/NightscoutFoundation/xDrip/assets/111485771/3772464e-d3e9-4131-843d-f8a1c2c18e46)

**Comment by Navid200** [comment]
I mean something like this: https://www.gsmarena.com/oppo_watch_4_pro-12532.php

Except, that shows ColorOS Watch 3.0 - China

**Reply by SimonXuku** [reply]
I can't find anymore about this,but it's same as oppo watch2 and oppo watch3,I tested on Chinese version watch2 (android 8.1)and watch3(coloros V5.0 android 11),it also not work.

**Reply by Navid200** [reply]
These are the recommended settings for G6: https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html

**Reply by SimonXuku** [reply]
I'll try.thx
